docs/source/user/creating_an_environment.rst

```diff
@@ -1,11 +1,13 @@
 Creating an environment
 -----------------------
-Due to version incompatibilities, setting up a full environment can be somewhat
-difficult to navigate, and it is recommended that you follow these steps.  This
-guide assumes that you already have anaconda set up on your computer.  If you do
-not, please follow `these instructions <https://www.anaconda.com/products/distribution>`_.
+.. code-block:: bash
 
     conda create -n csm python=3.6
     conda activate csm
     conda install -c conda-forge csmapi
     conda env update -n csm -f environment.yml
+
+Due to version incompatibilities, setting up a full environment can be somewhat
+difficult to navigate, and it is recommended that you follow these steps.  This
+guide assumes that you already have anaconda set up on your computer.  If you do
+not, please follow `these instructions <https://www.anaconda.com/products/distribution>`_.
```

Comment:
This link doesn't seem to go to any instructions? In the past I've directed people to the miniforge instructions: https://github.com/conda-forge/miniforge

Reply:
addressed in latest commits.